kubernetes/structure_pod_security_policy_spec.go

```diff
@@ -0,0 +1,658 @@
+package kubernetes
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+
+	v1 "k8s.io/api/core/v1"
+	v1beta1 "k8s.io/api/extensions/v1beta1"
```

Comment:
For PodSecurityPolicy we want to use `"k8s.io/api/policy/v1beta1"` as it has been removed from `extensions` in kubernetes v1.18.

Reply:
Fix